external-docs/docs/quick-start/qube config.md

```diff
@@ -0,0 +1,239 @@
+| `range_uri`      | A uri which describes the range of the values within the column (Optional) | *none*                                                       |
+| `uri_override`   | Override the uri created automatically for the column (Optional) (Advanced) | `tidy_data.csv#uri_safe_column_header_from_csv`              |
+| `uri_template`   | Override the uri generated for values within the uri (Optional) (Advanced) | **TODO** Look this up                                        |
+| `codelist`       | Link to an existing code list (Optional) (Advanced)          |                                                              |
+
+## Attributes
+
+| **field name**   | **description**                                              | **default value**                                            |
+| ---------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| `type`           | The type of the column (Required)                            | *dimension*                                                  |
+| `label`          | The title of the column (Optional)                           | The capital case of the header in the csv file with spaces replacing underscores |
+| `description`    | A description of the contents of the column (Optional)       | *none*                                                       |
+| `from_existing`  | The uri of the resource for reuse/extension (Optional)       | *none*                                                       |
+| `definition_uri` | A uri of a resource to show how the column is created/managed (i.e. a uri of a PDF explaining a list of units) (Optional) | *none*                                                       |
+| `data_type`      | The xml data type of the contents of the column, if this is provided it becomes a Literal Attribute column (Optional) | *none*                                                       |
+| `required`       | If this boolean value is true csvcubed will flag to the user if there are blank values in this column | *none*                                                       |
+| `codelist`       | Link to an existing code list (Optional) (Advanced)          | *none*                                                       |
+
+## Observations
+
+Observations are the most important component of a CSV-W data set. Observation columns can have measures and units defined against them to obviate the need for separate unit and measure columns in a single unit/measure data set.
+
+| **field name** | **description**                                              | **default value** |
+| -------------- | ------------------------------------------------------------ | ----------------- |
+| `data_type`    | The data type of the observations. This can only be decimal or integer. (Optional) | *decimal*         |
```

Comment:
> This can only be decimal or integer

Is that strictly true?

Reply:
No, but for a quick start guide do we need to introduce that level of complexity?

Reply:
I'd alter it to say `this should generally be a decimal or integer`.

And tbh I'm not sure I'd call this a quick start guide. I'd say this is more indepth than we'd want for that and doesn't give the user a clear example to follow. Not that I think you should change what you've done here, I just don't think it's `quick start`.